info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-02T03:02:03.932Z"}
info: Connected to the database. {"timestamp":"2022-06-02T03:02:04.039Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-02T03:16:04.006Z"}
info: Connected to the database. {"timestamp":"2022-06-02T03:16:04.097Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-02T03:22:49.114Z"}
info: Connected to the database. {"timestamp":"2022-06-02T03:22:49.187Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-02T03:23:00.253Z"}
info: Connected to the database. {"timestamp":"2022-06-02T03:23:00.320Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-02T04:08:36.380Z"}
info: Connected to the database. {"timestamp":"2022-06-02T04:08:36.713Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-02T04:10:10.737Z"}
info: Connected to the database. {"timestamp":"2022-06-02T04:10:10.811Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-02T04:11:32.517Z"}
info: Connected to the database. {"timestamp":"2022-06-02T04:11:32.577Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-02T04:12:55.694Z"}
info: Connected to the database. {"timestamp":"2022-06-02T04:12:55.754Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-02T05:46:59.663Z"}
info: Connected to the database. {"timestamp":"2022-06-02T05:46:59.758Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-02T05:47:42.388Z"}
info: Connected to the database. {"timestamp":"2022-06-02T05:47:42.466Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-02T05:54:04.847Z"}
info: Connected to the database. {"timestamp":"2022-06-02T05:54:04.925Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-02T05:55:21.879Z"}
info: Connected to the database. {"timestamp":"2022-06-02T05:55:21.992Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-02T05:57:27.701Z"}
info: Connected to the database. {"timestamp":"2022-06-02T05:57:27.774Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-02T06:00:58.539Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-02T06:01:14.621Z"}
info: Connected to the database. {"timestamp":"2022-06-02T06:01:14.730Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-02T06:19:44.192Z"}
info: Connected to the database. {"timestamp":"2022-06-02T06:19:44.366Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-02T06:25:44.033Z"}
info: Connected to the database. {"timestamp":"2022-06-02T06:25:44.128Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-02T06:28:30.567Z"}
info: Connected to the database. {"timestamp":"2022-06-02T06:28:30.646Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-02T06:38:53.413Z"}
info: Connected to the database. {"timestamp":"2022-06-02T06:38:53.481Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-02T06:43:20.189Z"}
info: Connected to the database. {"timestamp":"2022-06-02T06:43:20.281Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-02T06:43:34.370Z"}
info: Connected to the database. {"timestamp":"2022-06-02T06:43:34.504Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-02T06:47:26.038Z"}
info: Connected to the database. {"timestamp":"2022-06-02T06:47:26.138Z"}
error: 
    status - 500 
    message - Error: User does not exist
    at _callee2$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/user.service.js:26:11)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/login 
    method - POST 
    IP - ::1
   {"timestamp":"2022-06-02T06:50:57.124Z"}
error: 
    status - 500 
    message - TypeError: Cannot read properties of undefined (reading 'sendEmail')
    at _callee3$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/user.service.js:44:16)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/forgotpassword 
    method - POST 
    IP - ::1
   {"timestamp":"2022-06-02T06:52:44.334Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-02T06:56:07.017Z"}
info: Connected to the database. {"timestamp":"2022-06-02T06:56:07.145Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-02T06:56:58.448Z"}
info: Connected to the database. {"timestamp":"2022-06-02T06:56:58.563Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-02T06:58:27.822Z"}
info: Connected to the database. {"timestamp":"2022-06-02T06:58:27.893Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-02T06:59:30.362Z"}
info: Connected to the database. {"timestamp":"2022-06-02T06:59:30.458Z"}
error: Invalid login: 535-5.7.8 Username and Password not accepted. Learn more at
535 5.7.8  https://support.google.com/mail/?p=BadCredentials d2-20020a630e02000000b003fa95b940bdsm2513360pgl.88 - gsmtp {"code":"EAUTH","command":"AUTH PLAIN","response":"535-5.7.8 Username and Password not accepted. Learn more at\n535 5.7.8  https://support.google.com/mail/?p=BadCredentials d2-20020a630e02000000b003fa95b940bdsm2513360pgl.88 - gsmtp","responseCode":535,"timestamp":"2022-06-02T06:59:48.690Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-02T07:10:07.704Z"}
info: Connected to the database. {"timestamp":"2022-06-02T07:10:07.821Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-02T07:12:31.755Z"}
info: Connected to the database. {"timestamp":"2022-06-02T07:12:31.850Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-02T07:13:14.940Z"}
info: Connected to the database. {"timestamp":"2022-06-02T07:13:15.046Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-02T07:16:20.365Z"}
info: Connected to the database. {"timestamp":"2022-06-02T07:16:20.439Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-02T07:16:44.753Z"}
info: Connected to the database. {"timestamp":"2022-06-02T07:16:44.865Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-02T07:17:02.045Z"}
info: Connected to the database. {"timestamp":"2022-06-02T07:17:02.154Z"}
error: 
    status - 500 
    message - Error: User does not exist
    at _callee3$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/user.service.js:41:11)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/forgotpassword 
    method - POST 
    IP - ::1
   {"timestamp":"2022-06-02T07:17:08.215Z"}
error: 
    status - 500 
    message - Error: User does not exist
    at _callee3$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/user.service.js:41:11)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/forgotpassword 
    method - POST 
    IP - ::1
   {"timestamp":"2022-06-02T07:17:23.976Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-02T07:18:31.607Z"}
info: Connected to the database. {"timestamp":"2022-06-02T07:18:31.677Z"}
error: Invalid login: 535-5.7.8 Username and Password not accepted. Learn more at
535 5.7.8  https://support.google.com/mail/?p=BadCredentials n22-20020aa78a56000000b0051b95c76752sm728658pfa.153 - gsmtp {"code":"EAUTH","command":"AUTH PLAIN","response":"535-5.7.8 Username and Password not accepted. Learn more at\n535 5.7.8  https://support.google.com/mail/?p=BadCredentials n22-20020aa78a56000000b0051b95c76752sm728658pfa.153 - gsmtp","responseCode":535,"timestamp":"2022-06-02T07:18:39.466Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-02T08:22:57.434Z"}
info: Connected to the database. {"timestamp":"2022-06-02T08:22:57.547Z"}
error: Invalid login: 535-5.7.8 Username and Password not accepted. Learn more at
535 5.7.8  https://support.google.com/mail/?p=BadCredentials q68-20020a632a47000000b003fcc510d789sm1499494pgq.29 - gsmtp {"code":"EAUTH","command":"AUTH PLAIN","response":"535-5.7.8 Username and Password not accepted. Learn more at\n535 5.7.8  https://support.google.com/mail/?p=BadCredentials q68-20020a632a47000000b003fcc510d789sm1499494pgq.29 - gsmtp","responseCode":535,"timestamp":"2022-06-02T08:23:27.240Z"}
